src/plugins/intel_gpu/src/kernel_selector/cl_kernels/bucketize_ref.cl

```diff
@@ -44,11 +44,15 @@ KERNEL(bucketize_ref)(const __global INPUT0_TYPE* input, const __global INPUT1_T
     const uint f = dim2 % OUTPUT_FEATURE_NUM;
     const uint b = dim2 / OUTPUT_FEATURE_NUM;
 
+    const uint out_index = GET_INDEX(OUTPUT, ORDER);
+
+#if IS_BOUNDARY_EMPTY
```

Comment:
Could you share the motivation of handling IS_BOUNDARY_EMPTY in a special way?

I think you can implement it to check input1 size and just call `output[out_index] = 0` if input1 size is 0. Then we won't need to handle it from host code and that will be simpler.

Reply:
When the second input is empty, input1 is filled with 0 and come into the kernel. So it needs to handle this case with the boolean variable like IS_BOUNDARY_EMPTY.